output_folder: "data/issue/converted_bin/"
file_idx: 1
start_frame: 0
frame_step: 1
playback_rate: 10.0
frame_id: "map"
loop_playback: true
publish_frame_points: true
use_pred_frames: true
publish_voxels: true
publish_polylines: true
publish_merged_polylines: true
publish_map_points: true
publish_converted_filtered: true
publish_converted_raw: true
